Dockerfile

```diff
@@ -65,15 +67,86 @@ ENV LD_LIBRARY_PATH="/opt/Python35/lib:${LD_LIBRARY_PATH}"
 # Install pip packages for Python 3.5.
 RUN /opt/Python35/bin/pip3.5 install six progressbar2==3.37.1 wheel
 
+
+# Download and install Python 3.6.
+RUN source /opt/rh/devtoolset-7/enable && \
+    mkdir -p /opt/Python36 && \
+    mkdir -p /tmp/python_download && \
+    pushd /tmp/python_download && \
+    wget -nv 'https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz' && \
+    tar -xzf Python-3.6.9.tgz && \
+    cd Python-3.6.9 && \
+    ./configure --prefix=/opt/Python36/ --enable-shared && \
+    make -s && \
+    make altinstall && \
+    popd && \
+    rm -rf /tmp/python_download && \
+    yum clean all
+# Update the library search path such that the so is found by python.
+ENV LD_LIBRARY_PATH="/opt/Python36/lib:${LD_LIBRARY_PATH}"
+# Install pip packages for Python 3.6.
+RUN /opt/Python36/bin/pip3.6 install wheel
```

Comment:
what is the reason that you only install wheel and not six and progressbar as for Python 3.5?

Reply:
we install it for 3.5 because we ship it. For 2.7, 3.6, ... we only need it to build our _preonpy bindings and to finally build the wheel

Reply:
👍